example/proto/example/v1/service.proto

```diff
@@ -16,13 +16,19 @@ message ExampleOutput {
 }
 
 service OneWay {
-  rpc NoInput(google.protobuf.Empty) returns (ExampleOutput) {
-  }
-  rpc NoOutput(ExampleInput) returns (google.protobuf.Empty) {
-  }
+  rpc NoInput(google.protobuf.Empty) returns (ExampleOutput);
+
+  rpc NoOutput(ExampleInput) returns (google.protobuf.Empty);
 }
 
 service TwoWay {
+  option (nexustemporal.v1.service).name = "example.v1.two-way";
+
   rpc Example(ExampleInput) returns (ExampleOutput) {
+    option (nexustemporal.v1.operation).name = "example";
+  }
+
+  rpc Ignored(google.protobuf.Empty) returns (google.protobuf.Empty) {
+    option (nexustemporal.v1.operation).disabled = true;
```

Comment:
Could use a test for the service options.

Basically want to test the full matrix:

```
service default | operation
-------------------------
unspecified | unspecified
unspecified | enabled
unspecified | disabled
enabled | unspecified
enabled | enabled
enabled | disabled
disabled | unspecified
disabled | enabled
disabled | disabled
```